### PR TITLE
(MAINT) prep for 0.3.0 release

### PR DESCRIPTION
**Pull request by cprice404**

### CODE REVIEW

**Comment by cprice404** [comment]
This is blocked on puppetlabs/trapperkeeper#257

File: project.clj [hunk 1]
```diff
@@ -34,16 +34,9 @@
                  ;; 'uberjar-exclusions' example toward the end of this file.
                  [org.jruby/jruby-stdlib "1.7.26"]
 
-
-                 [org.clojure/tools.logging "0.3.1"]
```
Comment (camlow325) [inline project.clj]: Looks like we'd get version `0.2.6` from clj-parent -- which is a lower version than what we would have had previously.  Not sure if the newer version was intentional / required here?

Comment (cprice404) [inline project.clj]: Probably just the latest version available when we created this repo.  Would be good to update it in the parent project but doesn't seem like a blocker.

**Comment by haus** [comment]
sam-thumbs-up

**Pull request merged**